lib/hello/accounts.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Hello.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Hello.Repo
+
+  alias Hello.Accounts.{User, Credential}
+
+
+  def list_users do
+    User
+    |> Repo.all()
+    |> Repo.preload(:credential)
+  end
+
+  def get_user!(id) do
```

Comment:
@JuanMaGiraldo when the bang `!` method is used?

Reply:
It is a naming convention. The difference between a method with `!` and one that does not have it is that the bang method will raise an exception if the function encounters an error, and the other will only return the error.